### DISCUSSION TITLE
Add "algorithm" to list of theorem types

### DISCUSSION THREAD

**Discussion by jeremy9959** [quarto-dev/quarto-cli#3605] [Feature Requests]
I'd like to suggest adding "Algorithm" to the list of theorem environments, with #alg- and @alg- tags.

**Comment by fulem** [comment]
I'm also interested.

**Comment by jeremy9959** [comment]
I submitted PR #3640 to address this.

**Reply by fulem** [reply]
Excellent! Hope it will be merged
